### ISSUE TITLE
[BUG] BSOD: fsm_types. cpp: 361

### ISSUE THREAD

**Issue by meermanr** [prusa3d/Prusa-Firmware-Buddy#3361]
(I have searched for this bug, and not found it)

**Printer type** - MINI

**Printer firmware version** - 5.1.0-alpha1+12820

**Original or Custom firmware** - Original

**Optional upgrades** - Filament runout sensor, E3D Revo, 0.6mm nozzle (brass)

**USB drive or USB/Octoprint** - USB

**Describe the bug**
Blue screen of death while uploading g-code via PrusaLink from PrusaSlicer:

```
BSOD
fsm_types.срр:361
FSM DESTROY: PrintPreview does
not exist. Called from void
marlin_server::_server_print_l
oo?р
(),/home/admin/workspace/RELEA

5.2.0-alpha     [A]
```

**How to reproduce**
2 of 2 attempts to upload this from PrusaSlicer (via PrusaLink, the LAN-based stuff) has caused a BSOD.

**Expected behavior**
Upload completes, g-code stored on USB Mass Storage, print begins.

**G-code**
[Spool holder - one sided triangle_0.6n_0.3mm_PLA_MINI_2h56m.gcode.zip](https://github.com/prusa3d/Prusa-Firmware-Buddy/files/12899786/Spool.holder.-.one.sided.triangle_0.6n_0.3mm_PLA_MINI_2h56m.gcode.zip)

**Crash dump file**
[dump_buddy.bin.zip](https://github.com/prusa3d/Prusa-Firmware-Buddy/files/12899787/dump_buddy.bin.zip)

**Video**
Photo of Blue Screen of Death:

![BSOD fsm_types  cpp: 361](https://github.com/prusa3d/Prusa-Firmware-Buddy/assets/318099/d1dc601e-ff01-4ff3-8736-9ea707c414c4)

**Comment by meermanr** [comment]
Note that copying the same G-Code file to the USB drive, inserting it into the MINI, and then loading it via the usual on-screen menus works, but lacks a preview picture.

**Comment by darraghbr** [comment]
Is this a duplicate of #3280 ?

**Comment by meermanr** [comment]
Yes, it seems to be, but I didn't find it because it lacked the actual error badge as searchable text.

**Comment by danopernis** [comment]
This particular type of BSOD should be permanently fixed since v6.0.0, fixed via https://github.com/prusa3d/Prusa-Firmware-Buddy/commit/99da8a4f253df34dc34f5f81f3607b94498ae96e
